zerver/tests/test_realm.py

```diff
@@ -103,6 +103,13 @@ def update_with_api(**kwarg):
         self.assertEqual(realm.create_stream_by_admins_only, False)
 
         # email address change disabled
+        set_up_db('name_changes_disabled', False)
+        realm = update_with_api(name_changes_disabled=True)
+        self.assertEqual(realm.name_changes_disabled, True)
+        realm = update_with_api(name_changes_disabled=False)
+        self.assertEqual(realm.name_changes_disabled, False)
+
+        # email address change disabled
```

Comment:
you should also verify that the setting actually prevents changing one's name (probably by patching the existing name change tests).